README.md

```diff
@@ -50,20 +50,49 @@ Before running the project, these services must be running and acessible:
 
 Let's examine how to run the project:
 
-### Option 1: [OpenShift](https://www.openshift.com/)
-
-You may use the templates in `openshift/templates/` and upload them to
-Openshift to run the project without any further configuration. The template uses two docker images:
-[`quarck/ruby25-openscap`](https://hub.docker.com/r/quarck/ruby25-openscap/) and [`centos/postgresql-96-centos7`](https://hub.docker.com/r/centos/postgresql-96-centos7/).
+### Option 1: Ephemeral Environment with [Bonfire](https://github.com/RedHatInsights/bonfire/)
 
 #### Prerequisites
 
-* [`ocdeployer`](https://github.com/bsquizz/ocdeployer)
+* [`oc`](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz)
+  ([docs](https://docs.openshift.com/dedicated/4/cli_reference/openshift_cli/getting-started-cli.html))
+* [`bonfire`](https://github.com/RedHatInsights/bonfire/)
+
+#### Setup
+
+1. Log in into the ephemeral cluster
+2. Set up bonfire configuration
+
+    To use local deployement confguration update the `~/.config/bonfire/config.yaml` as follows:
+    ```
+    apps:
+    - name: compliance
+      components:
+        - name: compliance
+          host: local
+          repo: ~/path/to/local/compliance-backend
+          path: clowdapp.yaml
+          parameters:
+            REDIS_SSL: 'false'
+    ```
 
 #### Deploy
 
+Deployment with gateway and routes
+
 ```shell
-ocdeployer -s compliance your_openshift_project
+bonfire deploy compliance gateway insights-ephemeral --no-remove-resources all --set-image-tag 'quay.io/cloudservices/compliance-backend=latest-new'
```

Comment:
`--set-image-tag` should not be required, since it's set in cicd-common: https://gitlab.cee.redhat.com/insights-platform/cicd-common/-/blob/master/bonfire_configs/ephemeral_apps.yaml#L34

Reply:
I guess that cicd-common is read only if the appsre config is being used. In case of a local config (as seen on the example), the bonfire would set `IMAGE_TAG` to shortened hash of current commit, hence the override. Anyhow, we can move the option to a note. Thanks!